local-cli/eject/eject.js

```diff
@@ -21,6 +23,10 @@ const fs = require('fs');
  *
  * - `name` - The short name used for the project, should be TitleCase
  * - `displayName` - The app's name on the home screen
+ * - `icon` - with three other properties, `ios`, `android`, `default`
```

Comment:
I think it would be good to start by explaining that icon can be set to a path string, and that will be used as the icon on every platform

For the deep configuration, please create an indented sub-list of options.